src/main.rs

```diff
@@ -32,6 +32,9 @@ struct Args {
     /// then the executable will be in `abc`.
     #[clap(short)]
     output_path: Option<String>,
+    /// Where to put intermediary files (only for OptimizeBrilLLVM mode)
+    #[clap(long)]
+    output_dir: Option<String>,
```

Comment:
nit: rename this to something less similar to output path, or just reuse output path